static/include/app/apis/generated/app.md

```diff
@@ -899,7 +921,8 @@ Get the logs associated with a specific machine {{< glossary_tooltip term_id="pa
 
 ```python {class="line-numbers linkable-line-numbers"}
 part_logs = await cloud.get_robot_part_logs(
-    robot_part_id="abc12345-1a23-1234-ab12-a22a22a2aa22", num_log_entries=20)
+    robot_part_id="abc12345-1a23-1234-ab12-a22a22a2aa22", num_log_entries=20
```

Comment:
if the closing parentheses is on a new line, generally you style it like thi
```suggestion
    robot_part_id="abc12345-1a23-1234-ab12-a22a22a2aa22",
    num_log_entries=20
```

Reply:
done